### PR TITLE
Add capabilities and data section of transportation electrification

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Migrate documentation on transportation electrification in the open source docs

### What the code is doing
N/A

### Testing
N/A

### Where to look
No change compared to our internal document

### Usage Example/Visuals
Screen Shot 2023-01-12 at 11 23 21 AM

### Time estimate
5min

### CODE REVIEW

**Review by BainanXia — APPROVED** [review]

File: docs/demand/transportation_electrification/data.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Capabilities and Data
+#####################
+The transportation electrification module calculates an estimate of the additional hourly electricity demand from the electrification of transportation vehicles for all U.S. urban and rural areas for years ranging from 2017-2050.  The hourly estimation builds upon data collected representing over half a million driving events. 
+
+Additional data sets allow for profiles to be generated across the following
+dimensions: 
+
++ 4 vehicle types (light-duty vehicles (LDV), light-duty trucks (LDT), medium-duty
+  vehicles (MDV), and heavy-duty vehicles (HDV)) 
++ 33 simulation years (2017-2050)
```
Comment (dmuldrew) [inline docs/demand/transportation_electrification/data.rst:10]: @BainanXia @danlivengood I don’t think a year range is right here? Rather four specific years:
``` Python
:param int projection_year: year that is being modelled/projected
 to, 2017, 2030, 2040, 2050.
```

Comment (danlivengood) [reply docs/demand/transportation_electrification/data.rst:10]: @dmuldrew technically you are correct at the moment.  However, what I want to explore via the Regional_scaling_factor csv creation process is whether NREL has projections for all years from 2017-2050 or just those four.  I believe they have all 33, and the only reason we only have 4 is that's the subset that Brian wrote up in Matlab.  With the Python process, assuming the data exists, we could do all 33.

Comment (rouille) [reply docs/demand/transportation_electrification/data.rst:10]: Keep it as is? We can go through the whole documentation in the final stage to make sure everything is correct.

Comment (danlivengood) [reply docs/demand/transportation_electrification/data.rst:10]: I say yes.

Comment (danlivengood) [reply docs/demand/transportation_electrification/data.rst:10]: Fyi, confirmed that NREL's data is indeed all 33 years:
"Scenario data from the Electrification Futures Study Scenarios of Electric Technology Adoption and Power Consumption for the United States report. Annual projections from 2017 to 2050 of electric technology adoption and energy consumption for five scenarios"
from https://data.nrel.gov/submissions/92

**Pull request merged**